Wrappers/Python/test/test_FBP.py

```diff
@@ -1,28 +1,24 @@
 # -*- coding: utf-8 -*-
-#  CCP in Tomographic Imaging (CCPi) Core Imaging Library (CIL).
```

Comment:
In the apache guidance https://www.apache.org/licenses/LICENSE-2.0.html
it reccomends: " that you include a file or class name and description of purpose on the same "printed page" as the copyright notice for easier identification within third-party archives."
We could consider doing this or leave the CCPi CIL line to address part of this?